mmv1/third_party/terraform/resources/resource_bigtable_gc_policy.go

```diff
@@ -103,7 +103,7 @@ func resourceBigtableGCPolicy() *schema.Resource {
 				Type:          schema.TypeString,
 				Optional:      true,
 				ForceNew:      true,
-				Description:   `If multiple policies are set, you should choose between UNION OR INTERSECTION.`,
+				Description:   `Deprecated in favor of 'gc_rules'. If multiple policies are set, you should choose between UNION OR INTERSECTION.`,
```

Comment:
```suggestion
				Description:   `NOTE: 'gc_rules' is more flexible, and should be preferred over this field for new resources. This field may be deprecated in the future. If multiple policies are set, you should choose between UNION OR INTERSECTION.`,
```

Reply:
I think we should have a stronger wording. I would like to mark these fields as deprecated. Users should not continue to use these fields because they will have unexpected behavior and that is something we will not be responsible for. Your recommendation seems to say one is better and either one is fine. It is not entirely true, they should not be using the legacy fields. Furthermore, the legacy fields in KCC are likely not working.

Reply:
> Users should not continue to use these fields because they will have unexpected behavior and that is something we will not be responsible for.

These are released fields in the provider- we need to continue to support them / be responsible for them working until removed. If we're going to deprecate the fields, we need to be able to support in-place transitions between the new and old fields at the time of the deprecation, and continue to support them through removal.

> Your recommendation seems to say one is better and either one is fine. It is not entirely true, they should not be using the legacy fields.

For better or for worse, this _must_ be the official stance of the provider until we fulfill the requirements to deprecate them.

> Furthermore, the legacy fields in KCC are likely not working.

That's not great for KCC, but we shouldn't make decisions that negatively affect Terraform users based on the experiences of KCC users.

Reply:
Continue this in an offline review/discussion: https://docs.google.com/document/d/1c4sOjIq3tqJUTs-nOifIkV-DkEfgX5Wm5_gPn8gJiGo/edit?resourcekey=0-znmuMjJJEXO4S7OSfPt4tw#

Reply:
Done. @rileykarson , we really prefer deprecating these fields.

We have gone through the discussion. I think they should be deprecated and they meet all the requirements. Please let me know if you strongly object. Thanks!